src/container/ControllerContainer.ts

```diff
@@ -148,23 +154,27 @@ export class ControllerContainer<T extends BaseAPIClient> {
       app_name: appName,
       user_config: userConfig,
     };
-    return this.fetchAPI("POST", `${this.CONTROLLER_URL}/userconfig/set`, {
-      // FIXME: JSONObject has problem with ?
-      json: payload as any,
-    });
+    return this.fetchAPI(
+      "POST",
+      `${this.CONTROLLER_URL}/apps/${appName}/userconfigs`,
+      {
+        // FIXME: JSONObject has problem with ?
+        json: payload as any,
```

Comment:
Not sure what's the problem? It seems okay when I removed `as any`.

Reply:
I think we problem mentioned in the comment is invalid now, updated!